react/components/FilterBar/FilterTag.js

```diff
@@ -185,9 +185,7 @@ class FilterTag extends PureComponent {
                       <span className="fw5">{getFilterLabel()}</span>
                     ) : (
                       <Fragment>
-                        <span className="">{`${
-                          options[subject].label
-                        }:\xa0`}</span>
+                        <span className="">{`${options[subject].label}:\xa0`}</span>
```

Comment:
Remove empty `className`. Maybe the removal of the entire `span` tag should be better, I'm not sure.

Reply:
Well... It was just an eslint fix, so... I'll remove the `className` but I won't remove the `span` tag because I don't know what it's about